### PR TITLE
Localization support in HTML docs, simplification of checklinks.sh

### PR DESCRIPTION
**Pull request by cakrit**
Related to #5197 
Add support for translations of our documentation located in project https://github.com/netdata/localization
It works and you can check it out by adding a `/zh_CN` or `/test` on the home page of the generated HTML documentation. 
TODOs
- [x] Support localization in the html build process
- [x] Show language selection menu
- [x] Get the Chinese content additional pages into the localization project
- [x] Have PRs in `netdata/localization` automatically trigger the `netdata/netdata` Netlify builds. The checks must pass before a PR is merged
- [x] Improve the language selection menu.

### CODE REVIEW

File: packaging/installer/UNINSTALL.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ A workflow for uninstallation looks like this:
 NETDATA_PREFIX="<installation prefix>"   # put what you used as a parameter to shell installed `--install` flag. Otherwise it should be empty
 NETDATA_ADDED_TO_GROUPS="<additional groups>"  # Additional groups for a user running netdata process
 ```
-3. Download [netdata-uninstaller.sh](https://github.com/netdata/netdata/blob/master/packaging/installer/netdata-uninstaller.sh) and run it as follows: `netdata-uninstaller.sh --yes --env <path_to_environment_file>`. The default `path_to_environment_file` is `/etc/netdata`, it's the location of the file `.environment` that is used by the uninstaller.
+3. Download [netdata-uninstaller.sh](netdata-uninstaller.sh) and run it as follows: `netdata-uninstaller.sh --yes --env <path_to_environment_file>`. The default `path_to_environment_file` is `/etc/netdata`, it's the location of the file `.environment` that is used by the uninstaller.
```
Comment (paulfantom) [inline packaging/installer/UNINSTALL.md]: Why is this changed in a PR regarding localization?

Comment (cakrit) [reply packaging/installer/UNINSTALL.md]: It was a test, it's been removed.

**Comment by cakrit** [comment]
Please review again gents.

**Comment by cakrit** [comment]
Pressing the red button, I've made some minor updates to the Chinese translation files as well.
